astroid/rebuilder.py

```diff
@@ -828,6 +828,19 @@ def visit_try(self, node, parent):
         elif node.handlers:
             return self.visit_tryexcept(node, parent)
 
+    def visit_annassign(self, node, parent):
+        """visit an AnnAssign node by returning a fresh instance of it"""
+        newnode = nodes.AnnAssign(node.lineno, node.col_offset, parent)
+        annotation = _visit_or_none(node, 'annotation', self, newnode)
+        newnode.postinit(self.visit(node.target, newnode),
+                         annotation,
+                         _visit_or_none(node, 'value', self, newnode),
+                         nodes.Const(node.simple,
```

Comment:
Mm, now that you mention this, we can probably call ``visit_const`` instead of building it on our own. Thanks for finding this.